app/src/main/java/com/douglasbruce/tasky/core/domain/formatter/NameFormatter.kt

```diff
@@ -0,0 +1,19 @@
+package com.douglasbruce.tasky.core.domain.formatter
+
+import javax.inject.Inject
+
+class NameFormatter @Inject constructor() {
+
+    fun getInitials(fullName: String): String {
+        // Split the full name into first and last name.
+        val (firstName, lastName) = fullName.split(' ')
```

Comment:
What if there's a middle name? And what if the user accidentally types two spaces between the name, such as `Philipp  Lackner`? In that case, your app will currently misbehave, since after splitting, the list will contain an empty string between first and last name.
I think this isn't in the requirements explicitly, but would be a very valid input based on them. In that case, I'd still only show the initials from the first and last name, but the logic to get these changes.

PS: This would be a really good class to write some tests for, since the logic seems simple, but contains lots of edge cases as you can see